packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -717,89 +731,116 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
       child = parent;
     }
 
+    // `rect` in the new intermediate coordinate system.
+    Rect rectLocal;
+    // Our new reference frame render object's main axis extent.
+    double localMaxExtent;
```

Comment:
should this be called mainExtend instead of max? (not sure what max is referring too....)